File: A1/A1-009/A1-009.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input score
    int midterm, finals;
    cin >> midterm >> finals;
    // Output
    int total = midterm + finals;
    cout << total << '\n';
    
    if(total >= 50) { cout << "pass"; }
    else { cout << "fail"; }
    return 0;
}

File: A1/A1-009/A1-009.py
# --------------------------------------------------
# File Name : A1-009.py
# Problem   : Pass
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input score
midterm = int(input())
finals = int(input())

# Calculate total score
total = midterm + finals

# Output
print(total)
if total >= 50:
    print("pass")
else:
    print("fail")
